source/command-monitoring/tests/unified/command.yml

```diff
@@ -0,0 +1,94 @@
+description: "commands"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client client
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+        - commandFailedEvent
+  - database:
+      id: &database database
+      client: *client
+      databaseName: &databaseName command-monitoring-tests
+  - collection:
+      id: &collection collection
+      database: *database
+      collectionName: &collectionName test
+
+initialData:
+  - collectionName: *collectionName
+    databaseName: *databaseName
+    documents:
+      - { _id: 1, x: 11 }
+
+tests:
+  - description: "A successful command"
+    operations:
+      - name: distinct
+        object: *collection
+        arguments:
+          fieldName: x
+          filter: {}
+    expectEvents:
+      - client: *client
+        events:
+          - commandStartedEvent:
+              command:
+                distinct: *collectionName
+                key: x
+              commandName: distinct
+              databaseName: *databaseName
+          - commandSucceededEvent:
+              reply:
+                ok: 1
+                values:
+                  - 11
+              commandName: distinct
+
+  - description: "A failed command"
```

Comment:
this was also `count`. I switched it to just be a copy of the "failed find" test from find.yml. I'm figuring we should delete one of them, but I left both in for now for discussion of whether there is any value in testing some different command here instead